File: cpp/bc1871.cpp
#include <iostream>

using namespace std;

int main()
{
	int fVal, sVal;
	string res;
	
	while(true)
	{
	    res = "";
	    int j = 0;
	    
		cin >> fVal >> sVal;
		
		if(fVal == 0 && sVal == 0) break;
		else
		{
		    string so = to_string(fVal+sVal);
		    
		    for(int i = 0; i < so.size(); i++)
		        if(so[i] != '0') res += so[i];
		  
		    cout << res << endl;
		}
	}
}

File: python/bc1871.py
while True:
    res = ""
    cVal = input().split(" ")

    fVal = int(cVal[0])
    sVal = int(cVal[1])

    if fVal == 0 and sVal == 0: break
    else:
        so = str(fVal+sVal)

        for c in range(0, len(so)):
            if so[c] != '0': res += so[c]

        print(res)